doc/en/api/1.0.0/datastores.yaml

```diff
@@ -290,22 +285,609 @@ paths:
         405:
           description: Method Not Allowed
 
+parameters:
+  dataStorePost:
+    name: dataStoreBody
+    description: |
+      The data store body information to upload.
+      
+      The contents of the connection parameters will differ depending on the type of data store being added.
+
+      - GeoPackage
+
+        Examples:
+        - application/xml:
+
+          ```
+          <dataStore>
+            <name>nyc</name>
+            <connectionParameters>
+              <database>file:///path/to/nyc.gpkg</database>
+              <dbtype>geopkg</dbtype>
+            </connectionParameters>
+          </dataStore>
+          ```
+
+        - application/json:
+
+          ```
+          {
+            "dataStore": {
+              "name": "nyc",
+              "connectionParameters": {
+                "entry": [
+                  {"@key":"database","$":"file:///path/to/nyc.gpkg"},
+                  {"@key":"dbtype","$":"geopkg"}
+                ]
+              }
+            }
+          }
+          ```
+
+        Connection Parameters:
+
+        | key | description | level | type | required | default |
+        | --- | ----------- | ----- | ---- | -------- | ------- |
+        | Primary key metadata table | The optional table containing primary key structure and sequence associations. Can be expressed as 'schema.name' or just 'name' | user | String | False | ` ` |
```

Comment:
Looks amazing, want to double check the use of spaces in the key:

Checking [JDBCDataStoreFactory](https://github.com/geotools/geotools/blob/master/modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStoreFactory.java#L131):

```
  /** Metadata table providing information about primary keys **/
    public static final Param PK_METADATA_TABLE = new Param("Primary key metadata table", String.class,
            "The optional table containing primary key structure and sequence associations. Can be expressed as 'schema.name' or just 'name'", false);
```

Yep, spaces.